sygnal/sygnal.py

```diff
@@ -52,18 +54,23 @@
 }
 
 
-class Sygnal:
-    def __init__(self, config, custom_reactor, tracer=opentracing.tracer):
+class Sygnal(object):
```

Comment:
Inheriting from object is redundant in Python 3.

```suggestion
class Sygnal:
```

It used to have an effect in Python 2.3--2.7, to opt into "new-style" rather than "old-style" classes. Details [here](https://docs.python.org/2/reference/datamodel.html#new-style-and-classic-classes) and [here](http://python-history.blogspot.com/2010/06/inside-story-on-new-style-classes.html) if you're really curious.

Everything is a new-style class in Python 3.